pkg/operator/operator.go

```diff
@@ -115,6 +120,19 @@ func (c serviceCertSignerOperator) Sync(obj metav1.Object) error {
 		return nil
 	}
 
+	serviceCAConfig, err := c.serviceCAOperatorConfigClient.Get("cluster", metav1.GetOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Does the CRD and CR for this exist on an install right now?  Tempted to say this should ignore all errors.